### ISSUE TITLE
Usage examples in readme not working

### ISSUE THREAD

**Issue by smarianimore** [phulst/node-tile38#39]
Hi everybody,
I'm trying to use your library to exploit tile38 services, but I encounter problems in following the documentation.

For instance, the following statement which follow the documentation in the README
`let query = client.intersectsQuery('Station').detect('inside','outside').bounds(44.64701352313015, 10.886374409919698, 44.66163938662867, 10.892356791848103);`
returns error
`Error: DETECT is not allowed when FENCE is not specified`

However, when I try to pass such option as per documentation
`let query = client.intersectsQuery('Station', { fence: true}).detect('inside','outside').bounds(44.64701352313015, 10.886374409919698, 44.66163938662867, 10.892356791848103);`
the same error happens.

I tried putting the option in the detect() function, but it is not recognised as parameter.

Finally, I tried using executeCommand() function as below:
`client.executeCommand('INTERSECTS Station BOUNDS 44.64701352313015, 10.886374409919698, 44.66163938662867, 10.892356791848103 FENCE DETECT inside, outside', { parseJson: false }).then(result => {
    console.dir(result);
}).catch(err => {
    console.error("Error: " + err);
});`
but I get 
`TypeError: client.executeCommand is not a function`

I'm stuck as I cannot rely on the documentation...can anyone help?

Thanks in advance :)

[EDIT] Tile38 nom package version:

`"tile38": {
      "version": "0.8.0",
      "resolved": "https://registry.npmjs.org/tile38/-/tile38-0.8.0.tgz",
      "integrity": "sha512-Z7n/bsqrFHIUd+wr43wVMEwDg4wedBOznFjIDPShKnadrnuuLDHDzGdT/51L4nnX+SmJLg2B6KIxKqkZe+HlJQ==",
      "requires": {
        "redis": "^3.0.0",
        "redis-parser": "^3.0.0"
      }
    }`

Tile38 command line server version: `tile38-server version: 1.25.5 (1a0c28ce)`

**Comment by phulst** [comment]
If you are trying to set up a live geofence you need to use the executeFence() function. You can see here for an example: 

https://github.com/phulst/node-tile38/blob/master/examples/geofence.js

Let me know if that helps.

**Comment by smarianimore** [comment]
Hi @phulst , thanks for your interest.
Unfortunately the error I mentioned occurs even before calling executeFence (which indeed would be my next statement), right when setting up the query with the instruction reported above (that is basically the same in the example you pointed to).

PS: I edited the OP with tile38 package version and version of command line server

**Comment by phulst** [comment]
what version of Tile38 and node library are you using? I tried an intersects query by adding the following code to examples/search.js: 

```javascript
intersection = client.intersectsQuery('eastbay').detect('inside').bounds(37.7935,-122.2661, 37.8254, -122.2491);
intersection.executeFence((err, results) => {
    if (err) {
        console.error(err);
    } else {
        console.log(results.id + ": " + results.detect + " intersection");
        // console.dir(results);
    }
});
```

When I run it (with 'node examples/search.js' ) it works fine and doesn't produce the errors you are getting.

**Comment by phulst** [comment]
I did just fix an issue with the executeCommand function that I've pushed to master. Once we can confirm that there are no other issues related to the intersectsQuery I can release a new version.

That said, I should mention that the executeCommand should be able to support any command _except_ for live fences. The latter open up a persistent connection to the server and work a bit different internally. I could look into adding support for this in a future version.
